examples/web-wagmi/src/components/header/Header.tsx

```diff
@@ -9,28 +9,32 @@ export function Header() {
 
   return (
     <header>
-      <h1>Lens SDK 🌿</h1>
-
-      <p>
-        Examples of each hook used demonstrating a possible integration with&nbsp;
-        <a href="https://wagmi.sh/">wagmi</a>.
-      </p>
-
       <div
         style={{
           display: 'flex',
           alignItems: 'center',
           justifyContent: 'space-between',
-          gap: '1rem',
-          position: 'absolute',
-          top: '1rem',
-          right: '1rem',
-          width: 'auto',
+          width: '100%',
```

Comment:
do we need `width: 100%` ?

Reply:
I can double check... this was before xmas.

Reply:
Removed here: https://github.com/lens-protocol/lens-sdk/pull/61/commits/7c92d07195fc6f8be041351d0bf7c0785cc93d3b